website/events/api/calendarjs/serializers.py

```diff
@@ -50,7 +50,7 @@ def _registration_info(self, instance: Event):
                 return _("You are registered for this event")
         # Optional registration possible
         elif instance.optional_registration_allowed:
-            return _("You can optionally register for this event")
+            return _("Registering for this event is optional")
```

Comment:
The other messages begin with "You ...." maybe it's good to keep this, or change all of the message if you think this copy is better

Reply:
I definitely thought the 'you can optionally register' was weird haha, that's why I changed it. Not really a problem if it is not the same right?